packages/shell/package.json

```
@@ -4,6 +4,10 @@
 	"private": true,
 	"main": "./src/index.ts",
 	"types": "./src/index.ts",
+	"exports": {
```

Biome (2.5.6)

[error] The key exports was already declared.

(lint/suspicious/noDuplicateObjectKeys)